PSC-ONBOARDING-OFFBOARDING.md

```diff
@@ -0,0 +1,171 @@
+alphabetical placing:
+
+```markdown
+|[Product Security](committee-product-security/README.md)|product-security|* [Jane Doe](https://github.com/jdoe), ACME Ltd<br>* [Private Mailing List](security@kubernetes.io)
+```
+#### kubernetes/security repository
+
+##### file: https://github.com/kubernetes/security/blob/master/OWNERS_ALIASES
+
+Add / remove the PSC member(s) github name from `security/OWNERS_ALIASES` to the
+existing `product-security-committee` field, according to the usernames
+alphabetical placing:
+
+```yaml
+aliases:
+  product-security-committee:
+    - jdoe
+```
+
+####  Add/remove from OpsGenie rotation
+
+##### url: https://app.opsgenie.com/teams/dashboard/633ac733-665f-4d62-871e-cbc011f37037
+
+Once the user account has been created in OpsGenie, they must add a
+`github=GITHUB_USERNAME` tag to their OpsGenie profile for the tool to work.
+
+See the below example.
+
+![User Configuration](/images/user-tag.png)
+
+####  Add/ Remove PSC Owner permission from Kubernetes Security Mailing Lists
+
+New PSC members are added to various security lists and upgraded to "owner" (or
+downgraded to member if they are leaving the PSC)
+
+To add members and assign the "owner" role:
+
+1. Visit the groups URL as a user authenticated as an existing "owner".
+2. Select the 'Manage group' or 'Manager Members' link.
+3. Within the left side panel, select 'Direct add member'.
+4. Enter the new PSC members email address
+5. Select the 'All members' link, or search for the new member in the search box.
+6. Select the new member and change the drop down role to "Owner"
+
+---
+
+**Existing Accounts**
+
+It may be the case, that the user has already joined the group / mailing list.
+If this is the case, steps 1 to 4 can be ignored and you can proceed to change
+the role of their existing account.
+
+---
+
+To downgrade existing owners to members:
+
+1. Visit the groups URL as a user authenticated as an existing "owner".
+2. Select the 'All members' link, or search for the existing owner in the search box.
+6. Select the owner and change the drop down role to "Member".
+
+#####  Kubernetes Security Mailing List add / remove `owner` role.
+
+###### url: https://groups.google.com/a/kubernetes.io/forum/#!forum/security
+
+##### Kubernetes Security Discuss Mailing List add / remove `owner` role.
+
+###### url: https://groups.google.com/forum/#!forum/kubernetes-security-discuss
+
+##### Kubernetes Security Discuss Private Mailing List add / remove `owner` role.
+
+###### url: https://groups.google.com/a/kubernetes.io/forum/#!forum/security-discuss-private
+
+##### Kubernetes Security Announce Mailing List
+
+###### url: https://groups.google.com/forum/#!forum/kubernetes-security-announce
+
+##### Kubernetes Distributors Announce Mailing List
+
+###### url: https://groups.google.com/a/kubernetes.io/forum/#!forum/distributors-announce
+
+#### Kubernetes Security Disclosure Github Issue Access
```

Comment:
Maybe add a TBD section about adding HackerOne access.

Reply:
yes, sounds good. Do we have this information anywhere?